library/alloc/src/collections/vec_deque/mod.rs

```diff
@@ -2976,9 +2963,9 @@ impl<T, A: Allocator> Extend<T> for VecDeque<T, A> {
 }
 
 #[stable(feature = "extend_ref", since = "1.2.0")]
-impl<'a, T: 'a + Copy, A: Allocator> Extend<&'a T> for VecDeque<T, A> {
+impl<'a, T: 'a + Copy, A: Allocator + 'a> Extend<&'a T> for VecDeque<T, A> {
```

Comment:
Since this is `T: Copy` I don't think the specialization needs `T: Clone`, which means the default impl can use `iterator.copied()` instead.